src/Swarm/Game/State.hs

```diff
@@ -988,7 +997,12 @@ initGameStateForScenario ::
   Maybe FilePath ->
   ExceptT Text IO GameState
 initGameStateForScenario sceneName userSeed toRun = do
-  g <- initGameState
+  (warnings, g) <- initGameState
+  unless (null warnings) $
+    except $
+      Left $
+        T.unlines $
+          map prettyFailure warnings
```

Comment:
Done.  If not for `fourmolu` I would normally put several of the functions on the same line.  It looks like `fourmolu` puts `.` chains in a straight column rather than staggered indents as for `$` chains.

Reply:
Yes, I don't mind `f $ g $ h $ x` so much when it's on a single line (though I personally still prefer using `.` as much as possible) but indeed, I had in mind the fact that `fourmolu` insists on doing a hanging indent for each `$`, but lines up `.` chains in a single column.